Remove alert forwarding not supported note

hjiawei opened the pull request:
This changeset removes alerts forwarding not supported notes in Calico Enterprise and Cloud v3.15, v3.16, and root level docs. Calico Enterprise do support alert forwarding. It also reformats `README.md` codeblocks.

Fix applies to CC docs 3.15 and 3.16

hjiawei committed:
Remove alert forwarding not supported note

This changeset removes alerts forwarding not supported notes in
Calico Enterprise and Calico Cloud root, v3.15 and v3.16 docs.
Calico Enterprise do support alert forwarding.  It also reformats
`README.md` code blocks.

bmckercher123 reviewed (approved):
Approved, but copy to 3.16 and 3.17 docs. And fix in CC 3.15/3.16 docs.

bmckercher123 reviewed (changes requested):
Copy to appropriate products/versions.

hjiawei commented:
> Approved, but copy to 3.16 and 3.17 docs.

@bmckercher123 The root level docs (3.16/3.17) is also modified. I have updated the description to reflect this.

bmckercher123 commented:
@hjiawei You'll need to update CC doc for 3.15/3.16.

hjiawei commented:
> @hjiawei You'll need to update CC doc for 3.15/3.16.

CC docs are also updated.

bmckercher123 reviewed (approved):
LGTM
